language-server/modules/langserver-core/src/test/resources/completion/worker/workerChannelSend.json

```diff
@@ -0,0 +1,16 @@
+{
+    "position": {
+        "line": 14,
+        "character": 16
+    },
+    "source": "worker/source/workerChannelInteraction.bal",
+    "items": [
```

Comment:
Same as above

Reply:
yes, only `chn` will be suffcient for this scenario IMO.